src/Blazor.Extensions.AzureMaps/Component/MapComponent.cs

```diff
@@ -51,11 +46,22 @@ public async Task ClearShapes()
             await this.MapService.ClearShapes();
         }
 
+        public async Task ClearTiles()
```

Comment:
Single like tasks should be like `public Task ClearTiles() => this.MapService.ClearTiles();`